# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mythtv/settings-pro.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- mythtv-0.16/settings.pro	2004-09-07 23:20:16.000000000 +0200
+++ mythtv-0.16/settings.pro.new	2004-11-22 00:28:04.802739208 +0100
@@ -34,86 +34,3 @@
     DEFINES += CONFIG_VIDEO4LINUX
 }
 
-# X11 support
-CONFIG += using_x11
-
-# Default Xv support
-CONFIG += using_xv
-EXTRA_LIBS += -L/usr/X11R6/lib -lXinerama -lXv -lX11 -lXext -lXxf86vm
-
-# IVTV (PVR-x50) support.  Disable if building on Windows.
-CONFIG += using_ivtv
-DEFINES += USING_IVTV
-
-# LCDProc support
-#DEFINES += LCD_DEVICE
-
-# Default audio output, OSS
-CONFIG += using_oss
-DEFINES += USING_OSS
-
-# Native ALSA support
-#CONFIG += using_alsa
-#ALSA_LIBS = -lasound
-
-# Native ARTS support
-#CONFIG += using_arts
-#ARTS_LIBS = -L/opt/kde3/lib -ldl -lartsc -lpthread
-#EXTRA_LIBS += -L/opt/kde3/lib -ldl -lartsc -lpthread
-#INCLUDEPATH += /opt/kde3/include
-# For Mandrake, use the following:
-#ARTS_LIBS = -ldl -lartsc -lpthread
-#EXTRA_LIBS += -ldl -lartsc -lpthread
-#INCLUDEPATH += /usr/include/artsc
-
-# DVB support
-#CONFIG += using_dvb
-#DEFINES += USING_DVB
-# Note: INCLUDEPATH should point to the directory with
-#   'linux/dvb/frontend.h', not the directory with frontend.h
-#INCLUDEPATH += /usr/src/linuxtv-dvb-1.0.1/include
-
-# Native lirc support
-#CONFIG += using_lirc
-#LIRC_LIBS = -llirc_client
-
-# XvMC support, modify as necessary.
-#CONFIG += using_xvmc
-#DEFINES += USING_XVMC
-#EXTRA_LIBS += -lXvMCNVIDIA -lXvMC
-
-# VIA cle266 support
-#CONFIG += using_viahwslice
-#DEFINES += USING_VIASLICE
-#EXTRA_LIBS += -lddmpeg
-
-# DirectFB support
-#CONFIG += using_directfb
-#EXTRA_LIBS += `directfb-config --libs`
-#QMAKE_CXXFLAGS += `directfb-config --cflags`
-
-# Windows support
-#CONFIG += win32
-#DEFINES += _WIN32
-
-# Mac OS X support
-macx {
-    # For source that uses Qt, Q_OS_MACX is defined.
-    # For source that uses config.h, CONFIG_DARWIN is defined.
-    # For other source (e.g. libmythtv), we define this
-    DEFINES += CONFIG_DARWIN
-}
-
-# DirectX support
-#CONFIG += using_directx
-#DEFINES += USING_DIRECTX
-
-# OpenGL support for vertical retrace sync
-#DEFINES += USING_OPENGL_VSYNC
-#EXTRA_LIBS += -lGL -lGLU
-#CONFIG += using_opengl
-
-# Allow use of XrandR to change display resolutions
-#CONFIG += using_xrandr
-#DEFINES += USING_XRANDR
-
